closure/goog/net/jsloader.js

```diff
@@ -94,36 +94,26 @@ goog.net.jsloader.scriptsToLoad_ = [];
  * @param {Array<string>} uris The URIs to load.
  * @param {goog.net.jsloader.Options=} opt_options Optional parameters. See
  *     goog.net.jsloader.options documentation for details.
+ * @return {!goog.async.Deferred} The deferred result, that may be used to add
+ *     callbacks
  */
 goog.net.jsloader.loadMany = function(uris, opt_options) {
   // Loading the scripts in serial introduces asynchronosity into the flow.
-  // Therefore, there are race conditions where client A can kick off the load
-  // sequence for client B, even though client A's scripts haven't all been
-  // loaded yet.
-  //
-  // To work around this issue, all module loads share a queue.
+  // The client receives a deferred to manage parallel loads.
   if (!uris.length) {
-    return;
-  }
-
-  var isAnotherModuleLoading = goog.net.jsloader.scriptsToLoad_.length;
```

Comment:
I think the addition is reasonable. I wouldn't make it the default though. Adding another field to `opt_options` is probably the best way to do it.  Take a look at the `goog.net.jsloader.Options` type.

That said, can you do that in a separate PR? This returning deferred change seems valuable enough on its own.